### DISCUSSION TITLE
Demo For Badges Just Ignore

### DISCUSSION THREAD

**Discussion by atanasius13** [community/community#139974] [Other Feature Feedback, Questions, & Ideas]
Answer this question

**Comment by AtanasiusSuryaGunadharma** [comment]
For Badges

**Comment by AtanasiusSuryaGunadharma** [comment]
Tes

**Comment by t1noo7** [comment]
give me one pls :D

**Comment by ebndev** [comment]
Please see [this comment](https://github.com/orgs/community/discussions/139976#discussioncomment-10779892) for reference to why this discussion was closed and locked. Thanks!
